src/module.tsx

```diff
@@ -61,7 +61,6 @@ export const plugin = new PanelPlugin<StatPanelOptions>(StatPanel)
       category: mainCategory,
       settings: {
         options: [
-          { value: BigValueIconPosition.None, label: 'none' },
```

Comment:
any specific reason to remove this option?

Reply:
because there is none as an option inside the icon type if you want to hide it, another none option inside the icon position i think that is superficial